<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RevisionWindow</class>
 <widget class="QWidget" name="RevisionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Revise</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,8,1">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_TotalWords">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Microsoft YaHei UI&quot;;</string>
       </property>
       <property name="text">
        <string>Total</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_LeftWords">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Microsoft YaHei UI&quot;;</string>
       </property>
       <property name="text">
        <string>Left</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextBrowser" name="textBrowser_Text">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>5</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;Microsoft YaHei UI&quot;;</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
     <property name="spacing">
      <number>4</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButton_No">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>2</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 700 16pt &quot;Microsoft YaHei UI&quot;;</string>
       </property>
       <property name="text">
        <string>No</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Yes">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>2</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 700 16pt &quot;Microsoft YaHei UI&quot;;</string>
       </property>
       <property name="text">
        <string>Yes</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../asset/picture.qrc"/>
 </resources>
 <connections/>
</ui>
